### ISSUE TITLE
The form element remain invalid

### ISSUE THREAD

**Issue by bogdanalexe90** [danialfarid/ng-file-upload#1050]
Hi, i have a setup like this:
```html
    <div class="aw-file-select"
         name="fileUploadDrop"
         ng-model="vm.model"
         ng-model-options="{allowInvalid: true}"
         ng-disabled="vm.ngDisabled"
         ng-required="vm.ngRequired"
         ngf-drop
         ngf-allow-dir="false"
         ngf-change="vm.onFileChange($files, $file, $newFiles, $duplicateFiles, $event)"
         ngf-drag-over-class="vm.dragOverClass"
         ngf-validate="vm.validationObj"
         ngf-drop-disabled="vm.ngDisabled"
         ngf-drop-available="vm.dropSupported">

        <label class="aw-file-select-text" ng-show="vm.dropSupported"><span>Drop file here to upload</span></label>

        <button type="button" class="btn btn-default btn-sm aw-file-select-btn"
                ngf-select
                name="fileUploadSelect"
                ng-model="vm.model"
                ng-model-options="{allowInvalid: true}"
                ngf-change="vm.onFileChange($files, $file, $newFiles, $duplicateFiles, $event)"
                accept="{{vm.accept}}"
                ngf-validate="vm.validationObj"
                ngf-select-disabled="vm.ngDisabled"
                ng-disabled="vm.ngDisabled">
            <span ng-show="vm.model || !!vm.hasExternalFile">Change file</span>
            <span ng-hide="vm.model || !!vm.hasExternalFile">Select file</span>
        </button>
    </div>
```

When i drag an invalid file in the first div the form control becomes invalid. Then i select a valid file pressing the button but the first form control remains still invalid. i tried calling `$validate()` on file change but it doesn't seem to work.

Did you took in the account running the validation again when the model change from outside? 

Thanks for your help.

**Comment by danialfarid** [comment]
Can you create a jsfiddle? I don't quiet understand what validation you are talking about.

**Comment by bogdanalexe90** [comment]
This was fixed in `9.0.9`. Sorry for delayed answer.
